content/components/page-header.mdx

```diff
@@ -299,6 +299,74 @@ The title bar can include a static page title, actions, and leading and trailing
+
+- Only utilize `header` if one does not currently exist on the page.
+- Utilize `header` if the `PageHeader` consists of site-oriented content, typically existing at the top of the page.
+- Ensure that when used on `PageHeader`, that the component itself is not within other landmarks, such as `main`.
+
+#### PageHeader with `nav` landmark
+
+Navigation landmarks are best utilized to group links that pertain to navigation of the page. 
+You can have multiple navigation landmarks, but it’s important to ensure that each unique navigation element has a unique label attached, either via `aria-label` or `aria-labelledby`. 
+You can utilize `PageHeader.Navigation` as a `navigation` landmark through the `as` prop. 
+This further extends the accessibility of this component, as it allows users to navigate via landmarks easier.
+It’s important to note that if you utilize a component with an existing `navigation` landmark such as `UnderlineNav` then you should not use `PageHeader.Navigation` as a navigation landmark, as this would be a duplicate.
+
+For navigation, you should follow these guidelines: 
+- If you have a list or grouping of links within `PageHeader.Navigation`, you should utilize the `nav` type for this element.
+- If you’re already utilizing a component with a navigation landmark (such as `UnderlineNav`), you should not use `PageHeader.Navigation` as a navigation landmark.
+- If more than one `navigation` landmark exists on the page at a given time, you should add a unique label to each one.
+You can do this by either passing an `aria-label` or `aria-labelledby` attribute to `PageHeader.Navigation`.
+
+### Heading Structure
+
+`PageHeader.Title` renders its content as a heading in the background and most of the time it will have crucial information about the page, so it’s important to ensure that you’re using it in an accessible manner.
+
+`PageHeader.Title` is shipped as an `H2` by default, but only if you do not pass a heading level to it via the `as` prop. You should modify the level of this component if the default heading `H2` does not make sense with the existing heading structure on the page.
+
+### Dom Ordering
+
+When PageHeader has a context area and/or a leading action, the dom ordering of the component needs to be carefully considered to ensure that the page header is accessible to all users.
+To be able to achieve this, we need to make sure that all of the interactive content in the page header comes after the main heading of the page (PageHeader.Title).
+With this approach, we can ensure that the screen reader users who are navigating through the heading menu don't miss any of those interactive content.
+
+Note that the visual and DOM ordering of the elements can only differ if the page heading never contains one or more focusable elements. If the heading includes any focusable elements, the visual and the DOM order must match.
+
+The recommended dom order of the page header components is:
+
+```jsx
```

Comment:
This is too React specific because I didn't know how to convey the dom ordering in another way. I am open to feedback

Reply:
I'm not sure this belongs in the design guidelines. Maybe add this to `PageHeader.mdx` in primer/react instead.

Reply:
👍🏻 I'm removing this DOM ordering now entirely from the design guidelines. Since the mdx in primer/react will redirect eventually, I'll follow up with that if we think the example and feature stories don't communicate the usage of the API enough.